test/unit/rules/require-presentational-children-test.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'require-presentational-children',
+
+  config: true,
+
+  good: [
+    '<button></button>',
+    '<div></div>',
+    '<li role="tab">Tab title</li>',
+    '<li role="tab"><h3 role="presentation">Tab Title</h3></li>',
+    '<div role="button"><div><span></span></div></div>',
+    '<span role="checkbox"/>',
+    '<div role="article"><h2>Hello</h2></div>',
+    `
+    <ul role="tablist">
+      <li role="presentation">
+        <a role="tab" href="#">Tab 1</a>
+      </li>
+    </ul>
+    `,
+    `
+    <ul role="presentation">
```

Comment:
Let's take this one out, since we don't want our examples to be technically correct for the rule but otherwise promote something that is not best practice.

Reply:
👍